README.md

```diff
@@ -162,6 +162,32 @@ render() {
 }
 ```
 
+### How to use instanceRef with React Hooks?
+
+#### Import useRef 
+``` js
+import { useRef } from React;
+
+const instanceRef = useRef(null); 
+
+<EditorJs tools={EDITOR_JS_TOOLS} instanceRef={(instance) => (instanceRef.current = instance)} />
+```
+
+#### Save Editor Data
+```js
+let data = await instanceRef.current.save();
+```
```

Comment:
```suggestion
``` js
import { useRef } from React;

const instanceRef = useRef(null); 

async function handleSave() {
  const savedData = await instanceRef.current.save();
}

<EditorJs tools={EDITOR_JS_TOOLS} instanceRef={(instance) => (instanceRef.current = instance)} />
```
```

How about this?